automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/metadata/DataSpec.scala

```diff
@@ -22,33 +21,6 @@ class DataSpec extends FreeSpec with ParallelTestExecution with WebBrowserSpec w
   val methodConfigName: String = randomIdWithPrefix(SimpleMethodConfig.configName)
   val testData = TestData()
 
-  def makeMetadataFile(filePrefix: String, headers: List[String], rows: List[List[String]]): File = {
```

Comment:
Workspace Data table metadata testing moved to new spec called `MetadataImportSpec`